packages/scss/src/components/inputs/textfield/_textfield.material.scss

```diff
@@ -7,7 +7,7 @@
 		box-shadow: 0 1px 0 0 _component("textfield.border.color");
 		padding: {
 			left: 0;
-			right: 0;
+			right: 1.75rem;
```

Comment:
Why add this padding on the default textfield-input in material ?

Reply:
Oops, it was supposed to be in mod-search!
![image](https://user-images.githubusercontent.com/25581936/38742070-d44dc32e-3f3b-11e8-9d76-4df6fab9ac5b.png)

✔️